tensorboard/plugins/scalar/scalars_plugin.py

```diff
@@ -62,7 +62,11 @@ def get_plugin_apps(self):
 
   def is_active(self):
     """The scalars plugin is active iff any run has at least one scalar tag."""
-    return bool(self._multiplexer) and any(self.index_impl().values())
+    if not self._multiplexer:
+      return False
+
+    mapping = self._multiplexer.PluginRunToTagToContent(metadata.PLUGIN_NAME)
```

Comment:
nit: images_plugin.py just does this, which is a little more concise?  Or do you think that's too inscrutable?

return bool(self._multiplexer.PluginRunToTagToContent(metadata.PLUGIN_NAME))

Reply:
Awesome and concise. Lets use it. I verified that it exhibits short-circuiting behavior.